index.js

```diff
@@ -1,58 +1,81 @@
-let library = [];
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    this.updateDisplay(book);
+  }
+
+  removeBook(id) {
+    this.data = this.data.filter((bookObj) => bookObj.id !== id);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    window.location.reload();
+  }
+
+  updateDisplay(bookObj) {
+    const bookList = document.getElementById('book-list');
+    const book = document.createElement('LI');
+    book.innerHTML = `Title: ${bookObj.title} <br> Author: ${bookObj.author}`;
+    const deletetBtn = document.createElement('button');
+    deletetBtn.innerHTML = 'Delete';
+    deletetBtn.addEventListener('click', () => {
+      this.removeBook(bookObj.id);
+    });
+
+    book.appendChild(deletetBtn);
+    bookList.appendChild(book);
+  }
+
+  saver() {
+    this.data.forEach((book) => {
+      this.updateDisplay(book);
+    });
+  }
+}
+
+const library = new Library();
 
 function adder() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```